chatgpt-shell-google.el

```diff
@@ -231,58 +338,58 @@ For example:
-                                        (cond ((seq-first .content.parts)
-                                               (let-alist (seq-first .content.parts)
-                                                 .text))
-                                              ((equal .finishReason "RECITATION")
-                                               "")
-                                              ((equal .finishReason "STOP")
-                                               "")
-                                              ((equal .finishReason "CANCELLED")
-                                               "Error: Request cancellled.")
-                                              ((equal .finishReason "CRASHED")
-                                               "Error: An error occurred. Try again.")
-                                              ((equal .finishReason "END_OF_PROMPT")
-                                               "Error: Couldn't generate a response. Try rephrasing.")
-                                              ((equal .finishReason "LENGTH")
-                                               "Error: Response is too big. Try rephrasing.")
-                                              ((equal .finishReason "TIME")
-                                               "Error: Timed out.")
-                                              ((equal .finishReason "SAFETY")
-                                               "Error: Flagged for safety.")
-                                              ((equal .finishReason "LANGUAGE")
-                                               "Error: Flagged for language.")
-                                              ((equal .finishReason "BLOCKLIST")
-                                               "Error: Flagged for forbidden terms.")
-                                              ((equal .finishReason "PROHIBITED_CONTENT")
-                                               "Error: Flagged for prohibited content.")
-                                              ((equal .finishReason "SPII")
-                                               "Error: Flagged for sensitive personally identifiable information.")
-                                              (.finishReason
-                                               (format "\n\nError: Something's up (%s)" .finishReason))))
-                                      .error.message))))
-                    (unless (string-empty-p text)
-                      (setq response (concat response text)))
-                  (setq pending (concat pending
-                                        (or (map-elt chunk :key) "")
-                                        (map-elt chunk :value)))))
-              chunks)
-        (setq result
-              (list (cons :filtered (unless (string-empty-p response)
-                                      response))
-                    (cons :pending pending)))
-        result)
+        (let ((response)
```

Comment:
Looks like we got some rogue whitespace changes here. Let's revert these please.

Reply:
This happens automatically with apheleia. I am not sure why my standard is different than yours.